.circleci/config.yml.in

```diff
@@ -722,96 +722,6 @@ jobs:
             conda activate python${PYTHON_VERSION}
             python -c "import torchvision"
 
-  unittest_linux_cpu:
```

Comment:
This was forgotten in the PR that ported the CPU tests. Since `unittest_linux_cpu` was not listed in the `unittest` group (only visible in `config.yml`, but not in this template), this job wasn't run.